internal/federationin/config_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package federationin_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/google/exposure-notifications-server/internal/envconfig"
+	"github.com/google/exposure-notifications-server/internal/federationin"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestEnvconfigProcess(t *testing.T) {
```

Comment:
this is overkill - no reason to delete it if it's there, but testing for this belong w/ envconfig and each individual use shouldn't need extensive testing.

Reply:
Removed